content/components/comment-box.mdx

```diff
@@ -87,13 +87,15 @@ A label is shown above the comment box to establish the context and inform users
       role="presentation"
       width="456"
     />
-    <Caption>Don't hide the label. Also don't change the label to notify users of any kind of state or event, like when an error occurred.</Caption>
+    <Caption>Don't hide the label. Also don't use the label to notify users of any kind of state or event, like when an error occurred.</Caption>
   </Dont>
 </DoDontContainer>
 
-### Placeholder text
+### Placeholder text (recommended)
 
-The placeholder text is meant to give users addtional hints. Keep in mind that it will disappear as soon as users start typing. The default placeholder text is "Type here..."
+The placeholder text is meant to give users addtional hints. Keep in mind that it will disappear as soon as users start typing. The default placeholder text is "Type here...".
+
+Note: A placeholder text is not required, but still recommended. It helps to improve usablity and make sure users know where to type.
```

Comment:
Sounds good to me. 👍  The

> ...still recommended. It helps to improve usability and make sure users know where to type.

part was referring to https://github.com/primer/design/pull/298#discussion_r977339009. So even in cases where a placeholder text is not needed, it's recommended to have one just to make sure users realize that it's a `textarea` and they can type inside.

In https://github.com/github/github/pull/242242 I added a pattern where the placeholder text is just a short version of the label in case we can't come up with a better placeholder text (mention slash commands or so). E.g.

Label | Placeholder
--- | ---
Add a comment | Comment
Add a description | Description

![Screen Shot 2022-10-18 at 13 16 44](https://user-images.githubusercontent.com/378023/196334303-75523ff6-058a-4c06-a70e-2bcc589bca14.png)

Reply:
Ok, updated the default placeholder text from `Type here...` to `Comment`. Also kept the "usability" part.